docs/foundation/tokens/README.md

```diff
@@ -282,20 +282,87 @@ The below tables for the naming structure show that levels will be skipped when
 
 ### For Designers
 - A base requirement for accessing Cedar tokens and using them in designs and wireframes is that you are able to use Sketch version 53.2
-- Toolkits are platform specific. If designing for the web, use the UI Web Toolkit 
-- Tokens are available in Cedar’s UI Toolkit and found on the Tokens page 
+- Toolkits are platform specific. If designing for the web, use the UI Web Toolkit
+- Tokens are available in Cedar’s UI Toolkit and found on the Tokens page
 - When using tokens, it’s your responsibility to pass the entire token name to the developer during the design hand-off process—this can be through automated or manual methods
 
 <br>
 
 ### For Developers
 A base requirement for using Cedar tokens is that you are able to consume and maintain packages through the following development processes:
 - Web consumers:
-  - Your project can compile CSS variables 
+  - Your project can compile CSS variables
   - You can import NPM packages
 - Native iOS consumers using CocoaPods
 - Android consumers using Android package manager
 
+#### install
+
+To install the token package:
+
+`npm install --save-dev @rei/cdr-tokens@DESIRED_VERSION`
+
+Your project must be able to compile SCSS or LESS in order to make use of the SCSS and LESS modules.
+
+The token package contains `/dist/less` and `/dist/scss` folders, each of which contains `cdr-tokens.{less|scss}` files containing all the tokens in the corresponding format.
+
+Alternatively you could use `/dist/js` and/or `/dist/json` if your project cannot build SCSS and LESS.
+
+#### usage
+
+SCSS example:
+```
+@import '@rei/cdr-tokens/dist/scss/cdr-tokens.scss'; /* import the tokens file */
+
+.your-default-text-class {
+  /* use mixins to apply many properties at once */
+  @include cdr-text-default;
+}
+
+.your-other-text-class {
+  /* use scss variables if you need to apply specific properties */
+  color: $cdr-color-text-error-lightmode;
+}
+```
+
+LESS example:
+```
+@import '@rei/cdr-tokens/dist/less/cdr-tokens.less'; /* import the tokens file */
+
+.your-default-text-class {
+  /* use mixins to apply many properties at once */
+  .cdr-text-default();
+}
+
+.your-other-text-class {
+  /* use variables if you need to apply specific properties */
+  color: @cdr-color-text-error-lightmode;
+}
+```
+
+JS example:
+```
+const CdrTokens = require('@rei/cdr-tokens/'); /* import the tokens file */
```

Comment:
extra slash at the end of the import. Should also make sure this imports the esm version, as if you import the commonjs tokens object it loads that whole object into yr source build 😬 i `_think_` using the esm version should let you tree shake out the tokens you don't use

i think we should use a simpler/practical example here, maybe importing an easing function token and passing it to something, or grabbing a breakpoint and checking the window width against it.

Reply:
I purposely did not want the ESM version in this example because we do not have any other examples using the CJS version with `require()`.  If someone can use ESM, I don't see a scenario where they would even use the JS token package because they would likely have a more robust build system and would use SASS/LESS instead(Except maybe the breakpoint tokens).  As far as the example being too complicated, all the examples are applying the same text style, only the format is changing.  Showing that you have to apply 6 tokens in the JS format where only one is required in SASS/LESS is valuable IMO.  I will modify the require path to be more specific(`@rei/cdr-tokens/dist/js/cdr-tokens.common`) which I believe prevent excessive object size.

Reply:
I will also remove the for loop style application as I now think you are correct and it is overly complex.

Reply:
someone using ESM might still have reason to use the tokens in JS, for example to calculate a breakpoint or pass an easing function to an animation library. So we still need to document the ES style import, and should point out that the CJS import loads the entire tokens object into your built JS

I still think the JS example is too complex, I think an example using just 1 token would be much better (for example, checking the window size against one of the breakpoint tokens).Furthermore I don't think setting typography in JS is a good idea to document at all, even if it's to demonstrate that the SCSS is easier

Reply:
another example might be a 3rd party vue component which has an option to pass in color values or gutter sizes. 

i do see the value in documenting the same thing via each process, but if thats the case then we should use an example like checking a breakpoint or a single color value, rather than something complicated like typography (for example, scss/less could just set up a media query targeting a breakpoint, and the JS example could show checking the window size against that breakpoint)

Reply:
I removed the JS typography example and added breakpoint detection instead as per Cole's recommendation.  I left the SASS and LESS examples as they were as I see value in showing mixin and variable use.